ddtrace/contrib/botocore/patch.py

```diff
@@ -519,32 +519,17 @@ def patched_api_call(original_func, instance, args, kwargs):
                 try:
                     if "Messages" in result:
                         for message in result["Messages"]:
+                            pathway = None
```

Comment:
Should we move `pathway = None` out of the for loop (ie move it to line 521)

Reply:
we need to reset `pathway` to `None` for every message, we could have a case where some messages being received may have the `PROPAGATION_KEY_BASE_64` and others may not, and for messages that don't, we don't want other messages' `PROPAGATION_KEY_BASE_64` being used (rather just `None`)